markdown_lab_rs.py

```
@@ -7,16 +7,28 @@
 2. Build the extension: maturin develop
 """
 
+import json
 import logging
-from typing import List, Optional
+from enum import Enum
+from typing import Dict, List, Optional
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)